packages/drivers/form/src/actions.js

```diff
@@ -1,9 +1,9 @@
 export default state => form => ({
-  set: type => (values = {}) => state[type].addOrUpdate({ form, ...values }),
+  set: type => (values = {}) => state[type].addOrUpdate({ id: form, ...values }),
```

Comment:
I think this more readable with form as key

Reply:
I don't know, all reducers have a `name` or `id` in keys. 
Why change that ?
So, we use `id` to generate key, and here, i think it's more a `name`

Reply:
We are in `k-ramel` here, not in `metroscope`.

I think this is better to see : `form: 'measures'`
When the user do : `form('measures').something()` in the other hand

Reply:
I re-read the PR.
Maybe `name` is ok there.

BUT BUT: you can't put `name` since it can be a field key for the final user.

```
[name]
[email]
[phone number]
<submit>
```

I think we should add a word to avoid collisions: `k-form` ou `k-name`

Reply:
`@@form-name` ?

Reply:
I try with `formName` 
It's special enough to avoid collisions ?

Reply:
you are boring.
try `@@form-name` please